gen/gen.go

```
@@ -130,6 +130,8 @@ func Gen(exercise string, j interface{}, t *template.Template) error {
 		return fmt.Errorf(`didn't contain version: %v`, err)
 	}
 
+	classify(j)
```

don't ignore error here.